data/core/init.lua

```diff
@@ -173,11 +173,10 @@ local function project_scan_thread()
         if entries_count > config.max_project_files then
           core.project_files_limit = true
           core.status_view:show_message("!", style.accent,
-            "Too many files in project directory: stopping reading at "..
-            config.max_project_files.." files according to config.max_project_files. "..
-            "Either tweak this variable, or ignore certain files/directories by "..
-            "using the config.ignore_files variable in your user plugin or "..
-            "project config.")
+            "Too many files in project directory: stopped reading at "..
+            config.max_project_files.." files. For more information see "..
+            "usage.md at github.com/franko/lite-xl."
+            )
```

Comment:
After recompiling the error message stays the same. Is there some caching going on which I'm not aware of?

Reply:
Well, the problem may be the way you run lite-xl, the application may use the original lua files you installed. When you develop the best way to test is to use the script:

```sh
bash dev-utils/run-local <build>
```

In this way it will launch the new executable and you are sure it will used the modified lua files. In addition it will run in a dummy HOME directory to avoid interference with your install.

Reply:
Thanks this works well!

What if I want to change out the actual version I'm using. Can I delete those lua files?